src/AppBundle/Form/ContactType.php

```diff
@@ -12,6 +12,19 @@
 class ContactType extends AbstractType
 {
     /**
+     * @var bool
+     */
+    private $includeEmail = true;
```

Comment:
Maybe this could be made more explicit by subclassing to GeneralContactType? Just so we don't have boolean (constructor argument).